packages/stems/src/components/PillButton/PillButton.module.css

```diff
@@ -0,0 +1,42 @@
+.button {
+  height: unset !important;
```

Comment:
Using `!important` is very bad smell, are there other ways we could accomplish this? 😨